modules/lang-painless/src/yamlRestTest/resources/rest-api-spec/test/painless/30_search.yml

```diff
@@ -482,100 +482,3 @@
                         }]
     - match: { error.root_cause.0.type: "illegal_argument_exception" }
     - match: { error.root_cause.0.reason: "script score function must not produce negative scores, but got: [-9.0]"}
-
----
-
-"Flat-object fields from within the scripting":
```

Comment:
@mingshl apologies if I missed that, but why these test are removed?

Reply:
The flat object currently doesn't  support painless query yet. And it will be a future implementation.